plenoptic/simulate/models/naive.py

```diff
@@ -5,8 +5,8 @@
 import numpy as np
 from torch.nn import functional as F
 
-from plenoptic.tools.conv import same_padding
-from plenoptic.simulate.canonical_computations.filters import circular_gaussian2d
+from ...tools.conv import same_padding
```

Comment:
I switched to this because there can be some ambiguity with absolute imports: if you install `plenoptic` in an environment using pip, then clone the repo and try to use the local version, you'll have two versions of plenoptic, and absolute imports default to the pip-installed one (depends on your `sys.path`, I believe). I'm pretty sure this happened to me while working on something in pyrtools. This might only come up during development, but relative imports will always use the corresponding version, regardless of `sys.path` (and is more resilient to restructuring), so I think it's worth the slightly more confusing import statement